README.md

```diff
@@ -1,83 +1,40 @@
 [![Build Status](https://travis-ci.org/DataDog/dd-agent.svg?branch=master)](https://travis-ci.org/DataDog/dd-agent)
 
-The Datadog Agent faithfully collects events and metrics and brings
-them to [Datadog](https://app.datadoghq.com) on your behalf so that
-you can do something useful with your monitoring and performance data.
+# Important note
 
-You're looking at the source code right now. We provide a number of
-pre-packaged binaries for your convenience for both [Agent 5](https://app.datadoghq.com/account/settings?agent_version=5#agent) and [Agent 6]((https://app.datadoghq.com/account/settings#agent)). 
+This repository contains the source code for the Datadog Agent until major version 5.
+Although still supported, no major feature is planned for this release line and we
+encourage users and contributors to refer to the new Agent codebase, introduced
+with the release of version 6.0.0 and tracked in
+[a different git repository](https://github.com/DataDog/datadog-agent).
 
-See the [Datadog Agent Repo](https://github.com/DataDog/datadog-agent) for the source code for Agent 6. 
+## Changes
 
-Note: this repository does not contain the sources of the Trace Agent that
-collects traces for the APM feature. If you choose the installation from source
-or the Windows and Mac OS X builds you will need to install the Trace Agent
-separately. See
-[dd-trace-agent](https://github.com/DataDog/datadog-trace-agent) for
-instructions.
+Please refer to the [Change log](https://github.com/DataDog/dd-agent/blob/master/CHANGELOG.md)
+for more details about the changes introduced at each release.
 
-# [Change log](https://github.com/DataDog/dd-agent/blob/master/CHANGELOG.md)
+## Integrations
 
-# How to contribute code
+All checks have been moved to the [Integrations Core](https://github.com/DataDog/integrations-core) repo.
+Please look there to submit related issues, PRs, or review the latest changes.
 
-First of all and most importantly, **thank you** for sharing.
-
-If you want to submit code, please fork this repository and submit pull requests against the `master` branch.
-For more information, please read our [contributing guidelines](CONTRIBUTING.md).
-
-Please note that the Agent is licensed for simplicity's sake
-under a simplified BSD license, as indicated in the `LICENSE` file.
-Exceptions are marked with LICENSE-xxx where xxx is the component name.
-If you do **not** agree with the licensing terms and wish to contribute code nonetheless,
-please email us at <info@datadoghq.com> before submitting your
-pull request.
-
-# [Integration SDK](https://github.com/DataDog/integrations-core)
-
-All checks have been moved to the [Integration SDK](https://github.com/DataDog/integrations-core). Please look there to submit related issues, PRs, or review the latest changes.
-
-## Setup your environment
```

Comment:
not sure this section is documented anywhere else. Should we keep it in the README here? (we could repeat that development/contributions are encouraged on the Agent 6 repo instead of here, but it's good to keep the A5 base instructions for reference somewhere IMO)

Reply:
If this the only place we have these instructions they should be moved elsewhere, maybe the official docs.